crypto/build_dependency.sh

```diff
@@ -1,20 +1,27 @@
 #!/bin/bash
 
-set -euo pipefail
+PKG_DIR="./"
 
-# relic version or tag
-relic_version="9206ae50"
+# grant permissions if not existant
+if [[ ! -r ${PKG_DIR}  || ! -w ${PKG_DIR} || ! -x ${PKG_DIR} ]]; then
+   sudo chmod -R 755 ${PKG_DIR}
+fi
 
 rm -rf relic
 
+# relic version or tag
+relic_version="9206ae50"
```

Comment:
I would use the long SHA1, as this line has suddenly become a very important part of the build (granted, SHA1 is broken all on its own 🤦 )